### PR TITLE
DOI 887 migrate switchboard to gh

### PR DESCRIPTION
**Pull request by royki**
- Fixed Rsync
- Fixed Sentry

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+      - name: Prepare for non-master branch
+        if: github.ref != 'refs/heads/master'
+        run: |
+          cp src/environments/environment.dev.ts src/environments/environment.ts
+          echo "Prod Env Disabled"
+          npm run build
+
+      - name: Deploy on develop branch
+        if: github.ref == 'refs/heads/DOI-887_migrate_switchboard_to_GH'
```
Comment (dawidgil) [inline .github/workflows/deploy.yml:75]: i think here should be `refs/heads/develop`

Comment (royki) [reply .github/workflows/deploy.yml:75]: ohh, I forgot to remove that.

Comment (royki) [reply .github/workflows/deploy.yml]: fixed.

**Review by dawidgil — APPROVED** [review]
Lgtm. 
However i think it would be easier to fix the conflicts by creating a PR from new branch. This one was squashed and git history is different on develop and this branch. Do you want me to do that?

**Comment by royki** [comment]
> Lgtm.
> However i think it would be easier to fix the conflicts by creating a PR from new branch. This one was squashed and git history is different on develop and this branch. Do you want me to do that?

Hi, @dawidgil  Resolved the conflict. Now you can squash.

**Pull request merged**